packages/quantic/cypress/e2e/default-1/notifications/notifications.cypress.ts

```diff
@@ -1,41 +1,59 @@
 import {configure} from '../../../page-objects/configurator';
```

Comment:
We're transitioning to Playwright for e2e tests, so in order not to accrue technical debt, the right thing to do would be to write these new tests in Playwright:

https://coveo.slack.com/archives/G016TA2G485/p1717513178007719

Reply:
it will be part of a seperate ticket when we migrate the tests since the playwright set up is not yet done in quantic package